low-level/files/opt/seagate/sspl/setup/sspl_dev_deploy

```diff
@@ -229,8 +229,9 @@ class SSPLDevDeploy:
 
         try:
             with open("/etc/machine-id") as f:
-                node_key_id = f.read().strip("\n")
-                Conf.set("global_config", "cluster>server_nodes", {node_key_id:"srvnode1"})
+                machine_id = f.read().strip("\n")
+                Conf.set("global_config", "cluster>server_nodes", {machine_id:"srvnode-1"})
```

Comment:
cortx-provisioner is going to supply this. We have added n required keys set right?

Reply:
For single node provisioning, **sspl_dev_deploy** replicates provisioner work.
`We have added n required keys set right?` Whichever the value, sspl-ll service is capable of consuming the node_key_id mapped for its machine-id.

Reply:
Using machine-id we get node_key_id from ConfStore.
Using node_key_id we get respective enclosure_id.
So sspl-ll handles `n` required keys.

Reply:
Am asking that this key will be there in global config from provisioner?

Reply:
>So sspl-ll handles n required keys.

no sorry, had typo there. i meant
>cortx-provisioner is going to supply this. We have added **in** required keys set right?

Reply:
Yes, Chetan as per the latest provisioner sheet.

Reply:
Those keys are already added in global_config  sheet. So, we are using it.SSPL is just consume the available keys in global config and doesn't add or set keys.

sspl_dev_deploy script uses sample_global_config.yaml, in which Sandeep expects machine-id based on SSPL running environment like provisioner mainains global_config.

Reply:
we are good then, thanks.